src/es6/SelflessEntity.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/**
+ * Abstract Entity class to help create entity classes.
+ */
+export class SelflessEntity {
+	/**
+	 * Primes the object used by the entityFactory. Should never be called outside.
```

Comment:
I thought the intent was that it wasn't used by the entityfactory?

Reply:
Oops! I updated the params but not the initial comment. :(